src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const copy = {};
```

Comment:
Perhaps use a variable name that's more obvious. The function outputs an inverse rather than a copy.

Reply:
+1